src/platform/packages/shared/content-management/table_list_view_table/src/table_list_view.test.tsx

```diff
@@ -330,6 +331,85 @@ describe('TableListView', () => {
     });
   });
 
+  describe('column sorting with custom columns', () => {
+    const customSortingOptions: CustomSortingOptions = {
+      field: 'customColumn',
+      sortingLabels: [
+        {
+          label: 'Custom column A-Z',
+          direction: 'asc',
+        },
+        {
+          label: 'Custom column Z-A',
+          direction: 'desc',
+        },
+      ],
+    };
+
+    const setupCustomColumnSorting = registerTestBed<string, TableListViewTableProps>(
+      WithServices<TableListViewTableProps>(TableListViewTable, {
+        TagList: getTagList({ references: [] }),
+      }),
+      {
+        defaultProps: { ...requiredProps, customSortingOptions },
+      }
+    );
+
+    const hits: Array<UserContentCommonSchema & { field: string }> = [
+      {
+        id: '123',
+        updatedAt: twoDaysAgo.toISOString(),
+        type: 'maps',
+        attributes: {
+          title: '',
+        },
+        references: [],
+        field: 'customColumn',
```

Comment:
I think the test setup is incorrect. 
I think it should be: 
```suggestion
        customColumn: 'some-value',
```

We should also add a test that checks that custom option sorting actually works. If we'd done that, we'd realized that the test setup was faulty. Now we only check that custom sorting is displayed, but we don't check if it works.